src/seismometer/data/performance.py

```diff
@@ -217,8 +212,11 @@ def calculate_binary_stats(
     score_threshold_integer = int(score_threshold * 100)
     y_true = dataframe[target_col]
     y_pred = dataframe[score_col]
-    stats = calculate_bin_stats(y_true, y_pred, rho=rho)
-    return stats.iloc[100 - score_threshold_integer].round(5).to_dict()
+    stats = calculate_bin_stats(y_true, y_pred, rho=rho).drop("Threshold", axis=1)
+    stats = stats.iloc[100 - score_threshold_integer].round(5)
+    for name, percent in zip(COUNTS, PERCENTS):
```

Comment:
This seems unnecessarily fragile to assume order alignment... maybe we could generate the PERCENTS labels from COUNTS? Or do a test lookup to see if name+'%' is in percents?

Reply:
Updated the definition of `PERCENTS` to `[f"{count} (%)" for count in COUNTS]`